pennylane/interfaces/jax_jit.py

```diff
@@ -255,17 +255,17 @@ def wrapper(p):
 
         fwd_shape_dtype_struct = _extract_shape_dtype_structs(tapes, device)
 
-        jacobian_shape = [t.shape(device) + (len(p),) for t in tapes for p in params]
-        jac_dtypes = [_numeric_type_to_dtype(t.numeric_type) for t in tapes]
-
-        # Note: for qml.probs we'll first have a [1,dim] shape for the tape
-        # which is then reduced by the QNode
-        jacobian_shape = [
-            jax.ShapeDtypeStruct(tuple([shape]), dtype)
-            if isinstance(shape, int)
-            else jax.ShapeDtypeStruct(tuple(shape), dtype)
-            for shape, dtype in zip(jacobian_shape, jac_dtypes)
-        ]
+        # params should be the parameters for each tape queried prior, assert
+        # to double-check
+        assert len(tapes) == len(params)

```

Comment:
Great! Is there any test for this assertion?

Reply:
Added one. :+1: